discoursemap/modules/compliance_module.py

```
@@ -106,103 +106,103 @@ def __init__(self, scanner):
             self._test_data_retention()
-            
+
             # Breach Notification Testing
             self._test_breach_notification()
-            
-            print(f"[+] Compliance testing completed")
-            
+
+            print(f"{Fore.GREEN}[+] Compliance testing completed{Style.RESET_ALL}")
+
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)